ex/parser/nodes.py

```diff
@@ -8,6 +9,7 @@
 from Vintageous.ex.parser.tokens import TokenPercent
 from Vintageous.ex.parser.tokens import TokenSearchBackward
 from Vintageous.ex.parser.tokens import TokenSearchForward
+from Vintageous.state import State
```

Comment:
Using `State` here doesn't feel right. I think it'd be best to access the registers directly if we can. Not sure about that, though. Also, we'd need tests for this feature.